### ISSUE TITLE
Feature Request: Add ability to include anchor tag in tab

### ISSUE THREAD

**Issue by archer-jsonderman** [maslianok/react-responsive-tabs#63]
This is a feature request that is mainly for the mobile view of this, when the tabs turn to accordions. 

I would like to be able to prepend the tab content (title) with an anchor tag so that I can create jump links from a nav menu. Currently I don't see a way to inject a tag in the tab title to achieve this.

**Comment by tlehtimaki** [comment]
Would you be able to write pseudo code about what you would like to do? I could create a PR to see if it would be something feasible to do. 😊

**Comment by archer-jsonderman** [comment]
let me dig into it this weekend and see. I haven't looked into the src of this too much yet to see what all is going into create those tab items.

**Comment by archer-jsonderman** [comment]
Actually, I figured out how to do this without altering the source code. 

The title property can take array or a string (and an Object it looks like). Passing it an array will render each item in order. I passed an <a> tag in the first index, and the text of the tab in the second. Rendered as expected. 

`title:[<a name={props[program].tabName}/>,props[program].tabName]`

Maybe a way to wrap the contents of the tab in a block level tag, like a <p> would be nice. I haven't run this through any HTML validators to see if it barks are text being just in a DIV or not.
